ui/app/components/ui/text-field/text-field.component.js

```diff
@@ -24,7 +24,7 @@ const styles = {
   materialFocused: {},
   materialUnderline: {
     '&:after': {
-      borderBottom: '2px solid #f7861c',
+      borderBottom: `2px solid ${'$primary-1'}`,
```

Comment:
wait what? does this work?

Reply:
Nope, simply didn't push my hard-coded update.  Was left over from testing.

Reply:
aw man it doesn't lol. 😢